dapr/version.py

```diff
@@ -5,4 +5,4 @@
 Licensed under the MIT License.
 """
 
-__version__ = "0.8.0a0.dev"
+__version__ = "0.9.0a0.dev0"
```

Comment:
this should not include `0` because 0 is the build number which will be added by GH actions workflow automatically.